src/meeting/htmlCleaner.ts

```
@@ -0,0 +1,82 @@
+import { Page } from '@playwright/test'
+import { MeetingProvider, RecordingMode } from '../types'
+import { MeetHtmlCleaner } from './meet/htmlCleaner'
+import { TeamsHtmlCleaner } from './teams/htmlCleaner'
+
+export class HtmlCleaner {
+    private meetingProvider: MeetingProvider
+    private cleaner: MeetHtmlCleaner | TeamsHtmlCleaner | null = null
+    private isRunning: boolean = false
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)